packages/libtreesitter/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tree-sitter/tree-sitter
+TERMUX_PKG_DESCRIPTION="An incremental parsing system for programming tools"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.20.0
+TERMUX_PKG_SRCURL=https://github.com/tree-sitter/tree-sitter/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=4a8070b9de17c3b8096181fe8530320ab3e8cca685d8bee6a3e8d164b5fb47da
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make_install() {
+  make \
+    SONAME_MAJOR="0.20"\
+    SONAME_MINOR="0"\
```

Comment:
Do you need to set these?

Reply:
Without this the library files is named as `libtreesitter.so.0.0` or something like that and then there's a link `libtreesitter.so.0` and `libtreesitter.so`. With this it's named named as `libtreesitter.so.0.20.0`.

I'm not sure if it matters much.

Reply:
AFAIK these library versions are normally just used to show when the [ABI](https://en.m.wikipedia.org/wiki/Application_binary_interface) is changed, the version value normally does not correlate with the package version in any way so I think we can remove these lines

Reply:
Ok that makes sense . I'll remove it